pkg/secrethub/repo.go

```diff
@@ -14,6 +15,21 @@ type RepoService interface {
 	Delete(path string) error
 	// Get retrieves the repo with the given path.
 	Get(path string) (*api.Repo, error)
+	// EventIterator returns an iterator that retrieves all audit events for a given repo.
+	//
+	// Usage:
+	//  iter := client.Repos().EventIterator(path, &secrethub.AuditEventIteratorParams{})
+	//  for {
+	//  	event, err := iter.Next()
+	//  	if err == iterator.Done {
+	//  		break
+	//  	} else if err != nil {
+	//  		// Handle error
+	//  	}
+	//
+	//  	// Use event
+	//  }
+	EventIterator(path string, _ *AuditEventIteratorParams) AuditEventIterator
```

Comment:
Why do we have an unnamed parameter here? Would `params` be more descriptive?

Reply:
Yes, `params` as a name could also work.

The paramater is unnamed because it is not currently being used; there are no parameters to pass to the event iterator yet. The paramater is already added so that we can add parameters to the params struct without requiring a breaking change.